prometheus_client/openmetrics/exposition.py

```diff
@@ -94,24 +110,118 @@ def generate_latest(registry):
     return ''.join(output).encode('utf-8')
 
 
-def escape_metric_name(s: str) -> str:
+def escape_metric_name(s: str, escaping: str) -> str:
```

Comment:
Do we want defaults on this and `escape_label_name` as well? Technically this would break users if anyone else uses these functions, but they are pretty new so not as necessary compared to generate_latest.

Reply:
I don't see any public usage of it on Github so I'd say it is ok. We're still a 0.x library anyway.